exchanges/bitstamp/bitstamp_test.go

```diff
@@ -973,29 +984,52 @@ func TestOrderbookZeroBidPrice(t *testing.T) {
 
 func TestGetWithdrawalsHistory(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-	_, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
-	if err != nil {
-		t.Error(err)
+
+	if !mockTests {
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	}
+	h, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
+	assert.NoError(t, err, "GetWithdrawalsHistory should not error")
+	if mockTests {
+		assert.NotEmpty(t, h, "GetWithdrawalRequests should return a withdrawal request")
+		if mockTests {
```

Comment:
This mockTests check is redundant since it has already been checked before it

Reply:
removed